other/edge-server/client/README.md

```diff
@@ -374,8 +259,23 @@ that the Edge Server is running. If you have changed the URI and/or port where
 the Edge Server is listening for wireprotocol connections, change the URI
 and/or port in your `.env` file.
 
+## Run Multiple Edge Server Instances on a Host
```

Comment:
It looks like this section has gotten added in the middle of the "Troubleshooting" section. I see that Troubleshooting starts at ln 211 up above, and there's another "Troubleshooting" header at ln 291 below. I think _all_ of the troubleshooting errors listed here no longer apply in this version of the app, so I think we can just remove all of the troubleshooting details and keep the stuff between here and ln 289 below.

(And I think since you added the listener stuff, we can remove the part of ln 288 where we say "you can reload the app" - since you don't have to do that anymore, yay!)